package-manifest.toml

```diff
@@ -535,10 +535,10 @@ source.repo = "maghemite"
 # `tools/maghemite_openapi_version`. Failing to do so will cause a failure when
 # building `ddm-admin-client` (which will instruct you to update
 # `tools/maghemite_openapi_version`).
-source.commit = "de065a84831e66c829603d9a098e237e8f5faaa1"
+source.commit = "680d908445d3a7b58c1d69003a4df7b8ebab3e97"
```

Comment:
TODO before merging: point this back at a commit on `main`. This is currently referencing the https://github.com/oxidecomputer/maghemite/pull/182 branch.